doc/chapters/api/index.rst

```diff
@@ -79,6 +91,10 @@ Return Values
 #############
 * containerID: ``int32`` ID of created SoftwareContainer.
 
+Prerequisities
+##############
+None
```

Comment:
There is no dependency for the user to have done anything before calling this method. To have LXC installed etc is not what these headings are about.